Adding a troubleshooting section for ReplyURL

luleonpla opened the pull request:
Error AADSTS50011: The reply address ‘https://contoso.com’ does not match the reply addresses configured for the application*

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

luleonpla committed:
Adding a troubleshooting section for ReplyURL

Error AADSTS50011: The reply address ‘https://contoso.com’ does not match the reply addresses configured for the application*

NiyaNagi reviewed (approved):
Sweet!
